### DISCUSSION TITLE
is this bug ? cpu 0.0GHz

### DISCUSSION THREAD

**Discussion by ghost** [qtile/qtile#3180] [Q&A] [answered]
widget.CPU(
                background=colors[0],
                foreground=colors[3],           
                )
![2022-01-06-201546_1280x1012_scrot](https://user-images.githubusercontent.com/80429360/148423101-c42a9b45-3352-4388-94df-c81737fb1991.png)

**Comment by elParaguayo** [comment, marked as answer]
We're aware of this one. the `psutil` module has recently been upgraded and it's giving out different units to the previous version.

Someone's submitted a PR here for the issue: https://github.com/qtile/qtile/pull/3176 but I'm not very keen on it as it feels like an issue that should be fixed in `psutil`.

EDIT: I see a bug has been posted to the psutil developers so hopefully it will get fixed soon.

**Reply by ghost** [reply]
okay
